cypress/e2e/contexts/tag/specs/delete.cy.ts

```diff
@@ -0,0 +1,64 @@
+import { RegistrationForm, random_user_registration_data } from "../../user/registration";
+import { random_tag_name } from "../fixtures";
+
+describe("The admin user", () => {
+  let registration_form: RegistrationForm;
+
+  before(() => {
+    registration_form = random_user_registration_data();
+    cy.register_as_admin_and_login(registration_form);
+  });
+
+  after(() => {
+    cy.delete_user_from_database(registration_form.username);
```

Comment:
Hi @mario-nt, If the test fails, we could not have deleted the tag. Maybe we should try to delete it if it exists. If we want to keep the database clean after the test execution.